.github/workflows/chart-release.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com" 
 
       - name: Install Helm
-        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 #v3.5
+        uses: azure/setup-kubectl@3e0aec4d80787158d308d7b364cb1b702e7feb7f #v4.0.0
         with:
           version: 3.5.0 
```

Comment:
could you also increase the version to 3.12.* and test if it is still functional? We use that version in the devcontainer

Reply:
I have set up the correct azure/setup-helm; therefore, the version must be 3.5.0

Reply:
i did not mean the version of the GHA, but the Helmversion

Reply:
wilcard can not be set as version, i get this erreur:Downloading v3.12.*
  Error: Error: Failed to download Helm from location https://get.helm.sh/helm-v3.12.*-linux-amd[6](https://github.com/dBildungsplattform/schulportal-client/actions/runs/7894948487/job/21546529374#step:4:7)4.zip

Therefore, I tested Helm version 3.12.3, and the pipeline successfully ran
https://github.com/dBildungsplattform/schulportal-client/actions/runs/7895165196